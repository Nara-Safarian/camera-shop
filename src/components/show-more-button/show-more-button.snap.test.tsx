import { render } from '@testing-library/react';
import ShowMoreButton from './show-more-button';

test('ShowMoreButton snapshot should be rendered correctly', () => {
  const {container} = render(
    <ShowMoreButton onClick={function (): void {
      // nothing
    } }
    />
  );
  expect(container).toMatchSnapshot();
});
