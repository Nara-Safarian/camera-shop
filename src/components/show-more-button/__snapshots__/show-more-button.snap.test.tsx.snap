// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMoreButton snapshot should be rendered correctly 1`] = `
<div>
  <button
    class="btn btn--purple"
    type="button"
  >
    Показать больше отзывов
  </button>
</div>
`;
